cmd/scaffold.go

```diff
@@ -465,7 +465,7 @@ func (fnb *FlowNodeBuilder) initState() {
 		if fnb.RootBlock.ID() != rootBlockFromState.ID() {
 			fnb.Logger.Fatal().Msgf("mismatching root block ID, protocol state block ID: %v, bootstrap root block ID: %v",
 				fnb.RootBlock.ID(),
-				fnb.RootBlock.ID())
+				rootBlockFromState.ID())
```

Comment:
I think these should be reversed actually. `rootBlockFromState` is protocol state block ID, `fnb.RootBlock` is bootstrap block ID